packages/node/test/integration-setup.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const path = require('path');
+const ms = require('ms');
+const { version } = require('../package.json');
+const { intoChunks, NUMBER_OF_CHUNKS } = require('../../../utils/chunk-tests');
+
+const {
+  packAndDeploy,
+  testDeployment,
+} = require('../../../test/lib/deployment/test-deployment.js');
+
+jest.setTimeout(12 * 60 * 1000);
+
+module.exports = function setupTests(groupIndex) {
+  let builderUrl;
+  let builderUrlLastUpdated = 0;
+  const buildUtilsUrl = version.includes('canary') ? '@canary' : undefined;
+
+  beforeEach(async () => {
+    if (builderUrlLastUpdated < Date.now() - ms('30min')) {
+      const builderPath = path.resolve(__dirname, '..');
+      builderUrl = await packAndDeploy(builderPath);
+      builderUrlLastUpdated = Date.now();
+      console.log('builderUrl', builderUrl);
+    }
+  });
+
+  const fixturesPath = path.resolve(__dirname, 'fixtures');
+  const testsThatFailToBuild = new Map([
+    [
+      '45-noEmitOnError-true',
+      `index.ts(3,19): error TS2339: Property 'thisDoesNotExist' does not exist on type 'IncomingMessage'.\n`,
+    ],
+  ]);
+
+  let fixtures = fs.readdirSync(fixturesPath);
+
+  if (typeof groupIndex !== 'undefined') {
+    fixtures = intoChunks(NUMBER_OF_CHUNKS, fixtures)[groupIndex - 1];
+
+    console.log('testing group', groupIndex, fixtures);
+  }
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const fixture of fixtures) {
```

Comment:
It would be neat if we reversed the order so that tests in each chunk counted up with the chunks themselves.

Reply:
@ijjk Do you know why the tests are backwards? I'm pretty sure its happening before this line since the tests fixtures used to be in order so I suspect the chunking is backwards.

Reply:
The list tests currently relies on the order returned from `jest` here https://github.com/vercel/vercel/blob/afd303b94a3201c9bb6e11fe7d2c6f5825b4cb47/utils/chunk-tests.js#L17 

We could sort the tests alphabetically in jest if desired x-ref: https://jestjs.io/docs/configuration#testsequencer-string